### PR TITLE
Regen

### PR DESCRIPTION
**Pull request by sdroege**

### CODE REVIEW

**Comment by EPashkin** [comment]
There still warning in manual code
```
warning: unused import: `glib::translate::*`
 --> D:/eap/rust/0/gio\src\subprocess_launcher.rs:6:5
  |
6 | use glib::translate::*;
  |     ^^^^^^^^^^^^^^^^^^
  |
  = note: #[warn(unused_imports)] on by default
```

**Comment by sdroege** [comment]
> There still warning in manual code

I can't reproduce this here. How are you compiling?

**Comment by EPashkin** [comment]
I build examples\pending with uncommented defaults.
Or just use features="v2_40"

**Comment by sdroege** [comment]
>  Or just use features="v2_40"

Doesn't produce a warning for me either :( Can you fix it?

**Comment by sdroege** [comment]
> Doesn't produce a warning for me either :( Can you fix it?

Also removing that line will cause compile errors for me when using v2_40

**Comment by GuillaumeGomez** [comment]
Well then, I merge. :)

**Comment by EPashkin** [comment]
Seems in only appeared for windows, so it need be something like `#[cfg(any(all(feature = "v2_40", unix), feature = "dox"))]`,
we can fix this after all merged

File: src/auto/file.rs [hunk 1]
```diff
@@ -1429,44 +1393,42 @@ impl<O: IsA<File>> FileExt for O {
...
+        let callback = read_async_trampoline::<R>;
         unsafe {
-            ffi::g_file_read_async(self.to_glib_none().0, io_priority.to_glib(), cancellable.0, Some(callback), Box::into_raw(user_data) as *mut _);
+            ffi::g_file_read_async(self.as_ref().to_glib_none().0, io_priority.to_glib(), cancellable.map(|p| p.as_ref()).to_glib_none().0, Some(callback), Box::into_raw(user_data) as *mut _);
         }
     }
 
     #[cfg(feature = "futures")]
-    fn read_async_future(&self, io_priority: glib::Priority) -> Box_<futures_core::Future<Item = (Self, FileInputStream), Error = (Self, Error)>> where Self: Sized + Clone {
+    fn read_async_future<P: IsA<Cancellable> + Clone + 'static>(&self, io_priority: glib::Priority) -> Box_<futures_core::Future<Item = (Self, FileInputStream), Error = (Self, Error)>> where Self: Sized + Clone {
```
Comment (EPashkin) [inline src/auto/file.rs]: @sdroege Has trait constraint without usage :(
so examples still failing.

Comment (EPashkin) [reply src/auto/file.rs]: Added issue to rust: https://github.com/rust-lang/rust/issues/57660

Comment (sdroege) [reply src/auto/file.rs]: Yeah but that's not a problem by itself, only here we have the constraint but shouldn't have it :)

**Pull request merged**